src/meta/src/manager/catalog/mod.rs

```diff
@@ -103,6 +102,28 @@ where
     }
 }
 
+macro_rules! commit_meta {
```

Comment:
Seems all usages of `$meta_store` are `self.env.meta_store()`. Then we may further simplify the code by doing `$catalog_manager.env.meta_store()` in the macro, and when we use the macro, we only do `commit_meta!(self, ...)`

Reply:
Ok, it sounds better.